src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -1423,6 +1423,26 @@ static int patch_addr_in_movptr(address branch, address target) {
   return MOVPTR_INSTRUCTIONS_NUM * NativeInstruction::instruction_size;
```

Comment:
Maybe rename `MOVPTR_INSTRUCTIONS_NUM` as `MOVPTR1_INSTRUCTIONS_NUM`?
Or simply remove `MOVPTR_INSTRUCTIONS_NUM`:
`return 6 * NativeInstruction::instruction_size;  // lui + addi + slli + addi + slli + addi/jalr/load`

Reply:
We already have this defined in nativeinst, I used that enum.